pkg/manager/member/tikv_member_manager.go

```diff
@@ -588,31 +570,16 @@ func getTikVConfigMap(tc *v1alpha1.TidbCluster) (*corev1.ConfigMap, error) {
 		scriptModel.AdvertiseStatusAddr = "${POD_NAME}.${HEADLESS_SERVICE_NAME}.${NAMESPACE}.svc"
 		scriptModel.EnableAdvertiseStatusAddr = true
 	}
-	startScript, err := RenderTiKVStartScript(scriptModel)
+	cm, err := getTikVConfigMapForTiKVSpec(&tc.Spec.TiKV, tc, scriptModel)
 	if err != nil {
 		return nil, err
 	}
-	instanceName := tc.GetInstanceName()
```

Comment:
No need to keep the `ObjectMeta` here?

Reply:
we didn't set `ObjectMeta` in `getTikVConfigMapForTiKVSpec`, we set `ObjectMeta` after we call the `getTikVConfigMapForTiKVSpec` in each time.